requirements_test.txt

```diff
@@ -1,4 +1,4 @@
 -r requirements_dev.txt
 pytest-homeassistant-custom-component==0.5.13
-ocpp==0.8.3
-websockets==10.1
\ No newline at end of file
+ocpp==0.11.0
```

Comment:
Keep these two as >= so we can spot if an upstream issue causes testing to fail? If not then the manifest and requirements file can be bumped to the new version?